src/TodoModel.cpp

```diff
@@ -0,0 +1,62 @@
+#include "todo_lib/TodoModel.h"
+#include <string>
+#include <iostream>
+
+unsigned int TodoModel::nextId = 1;
```

Comment:
No, this doesn't look like a neat way to do unique Id

Reply:
How would you consider implementing this feature? And do elaborate on why this method isn't neat.
```suggestion
unsigned int TodoModel::nextId = 1;
```